### DISCUSSION TITLE
[求助]绳网等级怎么升啊

### DISCUSSION THREAD

**Discussion by EhuanyingK** [share121/inter-knot#1491] [其他]
有些人都4级了，到底怎么搞的
![114514](https://github.com/user-attachments/assets/7ab679b2-e591-4095-952d-b8a56d58f835)

**Comment by capacitor1** [comment]
这个委托我接了！

看：
#1467 
#1489

**Comment by EhuanyingK** [comment]
感谢

**Comment by Buer-Nahida** [comment]
我有16个Github公开仓库，然后我就16级了

**Comment by sfhjhsdfg** [comment]
在这里等级高的是大佬，~~我只负责跪~~
